backend/src/routes/auth/auth.router.ts

```diff
@@ -23,7 +23,15 @@ authRouter.get(`${PATH}/google/sign`, passportMiddleware.googleAuth);
 authRouter.get(
   `${PATH}/google/redirect`,
   passportMiddleware.googleMiddleware,
-  authController.signGoogle,
+  authController.signGoogleFacebook,
+);
+
+authRouter.get(`${PATH}/facebook/sign`, passportMiddleware.facebookAuth);
+
+authRouter.get(
+  `${PATH}/facebook/redirect`,
+  passportMiddleware.facebookMiddleware,
+  authController.signGoogleFacebook,
```

Comment:
Don't use the name of the exact 3rd party provider name if the controller isn't done exclusively for it. Because if we would decide to add Discord or Microsoft providers, the name would grow more and more. Use more generic name, like "sighOAuthProviders"